presto-product-tests/bin/run_on_docker.sh

```diff
@@ -159,6 +159,10 @@ trap terminate INT TERM EXIT
 # start hadoop container
 docker-compose -f "${DOCKER_COMPOSE_LOCATION}" up -d hadoop-master
 
+# start external database containers
+docker-compose -f "${DOCKER_COMPOSE_LOCATION}" up -d mysql
+docker-compose -f "${DOCKER_COMPOSE_LOCATION}" up -d postgres
```

Comment:
So this would start mysql and postgres containers every time, even if we are not running those tests. Not sure, if we want this.